domain/src/main/java/com/example/domain/utils/Const.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.domain.utils
+
+import com.example.domain.BuildConfig
+
+object Const {
+     const val BASE_URL = BuildConfig.BASE_URL
+     const val LOGIN_ENDPOINT = BuildConfig.LOGIN_ENDPOINT
+     const val CODE_401 = BuildConfig.CODE_401
```

Comment:
for code we should have separate const 

object Code{
     const val 401= ''
     const val 403 =''
     const val 500=''